scoap3/modules/tools/tools.py

```diff
@@ -39,74 +37,69 @@ def affiliations_export(country=None, year=None):
     result_headers = ['year', 'journal', 'doi', 'arxiv number', 'primary arxiv category',
                       'country', 'affiliation', 'authors with affiliation', 'total number of authors']
     result_data = []
-    index = 0
 
     # query ElasticSearch for result (and get total hits)
     query = get_query_string(country=country, year=year)
     search_results = current_search_client.search(
-        q=query, index=search_index, _source=source_fields, size=size, from_=index
-    )
+        q=query, index=search_index, _source=source_fields, scroll='1m')
```

Comment:
why the size is missing? it will get the default from ES which I believe it's `10`

Reply:
Also I would suggest to keep the scroll `5m`